### PR TITLE
add editorconf

### PR DESCRIPTION
**Pull request by tchakabam**
Editorconf. Useful to me, probably/hopefully to others too? :) 

http://editorconfig.org/

Let me know if there are things to add/change or if something conflicts with the lint rules.

:beers:

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+end_of_line = lf
+insert_final_newline = true
+
+indent_style = space
+indent_size = 2
+
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (towerz) [inline .editorconfig]: Is there any reason we can't trim whitespaces on `.md` files? If not, I'd rather keep trimming on.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+[{package.json}]
+indent_style = space
+indent_size = 2
```
Comment (towerz) [inline .editorconfig]: Just to remove some annoying tools complaints, I think it's better to add a new line at the end of the file.

File: .editorconfig (hunk 1 above)
Comment (tchakabam) [reply .editorconfig]: yes totally agree in fact

**Comment by tchakabam** [comment]
@towerz The config is incremental, it applies matches in order of occurence to files. So I was able to transform your suggestions by boiling it down a bit. Seems what we need is really simple. Let me know if that's it.

**Comment by towerz** [comment]
That's great, @tchakabam. Thanks!

**Pull request merged**